packages/lesswrong/components/ea-forum/EASurveyBanner.tsx

```diff
@@ -1,17 +1,18 @@
 import React, { useCallback } from "react";
 import { Components, registerComponent } from "../../lib/vulcan-lib";
-import { TypeformPopupEmbed } from "../common/TypeformEmbeds";
 import { useCurrentUser } from "../common/withUser";
 import { useTracking } from "../../lib/analyticsEvents";
 import { useCookiesWithConsent } from "../hooks/useCookiesWithConsent";
 import moment from "moment";
 import DeferRender from "../common/DeferRender";
+import { Link } from "@/lib/reactRouterWrapper";
+import { HIDE_EA_FORUM_SURVEY_BANNER_COOKIE } from "@/lib/cookies/cookies";
 
 const styles = (theme: ThemeType) => ({
   root: {
     position: "sticky",
     top: 0,
-    zIndex: 10000, // The typeform popup has z-index 10001
+    zIndex: 10, // The typeform popup has z-index 10001
```

Comment:
I think this is ok - I generally want all the header things to go on top of this banner, and I'm fine with the header covering the banner when the user scrolls up on the page